packages/cli/src/cmds/account.ts

```diff
@@ -169,14 +181,37 @@ class AccountLogin extends YargsCommand {
         description: "Authenticate with a specific Azure Active Directory tenant.",
         type: "string",
         default: "",
+      })
+      .options("service-principal", {
+        description: "Authenticate Azure with a credential representing a service principal",
+        type: "boolean",
+        default: "false",
+      })
+      .options("username", {
+        alias: "u",
+        description: "Client ID for service principal",
+        type: "string",
+        default: "",
+      })
+      .options("password", {
+        alias: "p",
+        description: "Client ID for service principal",
```

Comment:
Any plan to support certificate credential? Certificate is more commonly used than secret in production environments. And to meet security requirements as well.

Reply:
Add cert login logic to CLI.